src/components/Text.tsx

```diff
@@ -56,12 +59,62 @@ export function BasicText({
   return <p className={basicTextStyles({ bold, color, small })}>{children}</p>
 }
 
-const captionText = classnames(
+const captionText = (color: TTextColor) =>
+  classnames(
+    textColor(color),
+    textAlign('text-center'),
+    fontWeight('font-bold'),
+    fontSize('text-xl')
+  )
+export function CaptionText({
+  children,
+  color = 'text-blue-light',
+}: { color?: TTextColor } & ChildrenProp) {
+  return <p className={captionText(color)}>{children}</p>
+}
+
+interface InstructionTextProps {
+  primary?: boolean
+  small?: boolean
+  weight?: 'bold' | 'medium'
+}
+
+const instructionText = ({ primary, small, weight }: InstructionTextProps) =>
+  classnames(
+    textColor(primary ? 'text-blue-light' : 'text-secondary'),
+    fontWeight({
+      'font-bold': weight === 'bold',
+      'font-medium': weight === 'medium',
+    }),
+    fontSize({ 'text-sm': small }),
+    textAlign('text-center')
+  )
+export function InstructionText({
+  children,
+  primary,
+  small,
+  weight,
+}: InstructionTextProps & ChildrenProp) {
+  return (
+    <p className={instructionText({ primary, small, weight })}>{children}</p>
+  )
+}
+
+const codeText = classnames(
   textColor('text-blue-light'),
   textAlign('text-center'),
-  fontWeight('font-bold'),
-  fontSize('text-xl')
+  wordBreak('break-all'),
+  fontSize('text-xl'),
+  padding('p-2'),
+  cursor('cursor-pointer')
 )
-export function CaptionText({ children }: ChildrenProp) {
-  return <p className={captionText}>{children}</p>
+export function TokenText({
+  children,
+  onClick,
+}: ChildrenProp & { onClick: () => void }) {
+  return (
+    <code className={codeText} onClick={onClick}>
```

Comment:
Why `code`? It's bad for accessibility if we'll need it

Reply:
Removed